### PR TITLE
Add user database functionality

### PR DESCRIPTION
**Pull request by ErikssonF**

### CODE REVIEW

File: requests.rest [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+###
+
+POST http://localhost:3000/user/login 
+Content-Type: application/json
+
+{
+    "name": "Frege", "password": "password"
+
+}
```
Comment (LordRekishi) [inline requests.rest:21]: Do we need an extension to use this REST client?

**Review by LordRekishi — CHANGES_REQUESTED** [review]
Looks good, but:

1. Right now you can add multiple users with the same name, and with the same name + password. The latter resulting in them all getting the same ID. Maybe add a check and make sure the name added is not already in the database?

2. When we send in the wrong password in a login post we still get status 200, but a message saying "not allowed"

3. When we send in the wrong username in a login post we don't get anywhere, the program is just chugging along forever.

4. We need to add the user.db to .gitignore

Other than that, great start!

**Comment by LordRekishi** [comment]
[Account system setup](https://trello.com/c/KYfoNLPU/79-account-system-setup)

File: requests.rest (hunk 1 above)
Comment (ErikssonF) [reply requests.rest:21]: Yes, it's called REST Client. The file is not necessary for any functionality, it's just easier to make requests this way.

https://marketplace.visualstudio.com/items?itemName=humao.rest-client

**Review by LordRekishi — APPROVED** [review]

**Pull request merged**